diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index a63058238..03ac0010c 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v3.1.2.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v3.1.2/SVT-AV1-v3.1.2.tar.gz
-SVT-AV1.FETCH.sha256  = d0d73bfea42fdcc1222272bf2b0e2319e9df5574721298090c3d28315586ecb1
+SVT-AV1.FETCH.url     = https://github.com/Uranite/HandBrake-SVT-AV1-HDR/releases/download/contribs/svt-av1-hdr.tar.gz
+SVT-AV1.FETCH.sha256  = 1f6df71cf9826f73106d552321b9c563aa319916998fc2d477ca6218c1cdd7f0
 
 SVT-AV1.GCC.args.c_std =
 
